spec/index.html

```diff
@@ -1255,7 +1311,25 @@ <h3>Algorithm</h3>
           </details>
         </li>
         <li id="ca.7">Return the <a>serialized canonical form</a>
-          of the <a>normalized dataset</a>.</li>
+          of the <a>normalized dataset</a>.
+          Alternatively, return the <a>normalized dataset</a> itself,
+          which includes the <a>input blank node identifier map</a>,
+          and <a>canonical issuer</a>.
```

Comment:
Here's a suggestion to try and address the concerns here:

```suggestion
          Alternatively, or additionally, an implementation may return,
          upon request, the <a>normalized dataset</a> itself, which
          includes the <a>input blank node identifier map</a>,
          and <a>canonical issuer</a>.
          <p class="note">Technically speaking, one implementation
          might return a <a>normalized dataset</a> that maps
          particular blank nodes to different identifiers than another
          implementation, however, this only occurs when there are
          isomorphisms in the dataset such that a canonically serialized
          expression of the dataset would appear the same from either
          implementation.</p>
```